### ISSUE TITLE
How to mark a user's email as verified?

### ISSUE THREAD

**Issue by Nomango** [NationalSecurityAgency/skills-service#2033]
I opened the `skills.authorization.verifyEmailAddresses` function to prevent user registration (because I haven't configured email sending functionality). So I have to manually modify the root user as verified through email.

However, after I change the `user_attrs` table to email_verified=true, it always periodically reverts back to false. What could be the reason for this?

**Comment by sudo-may** [comment]
@Nomango thank you for the question! You are correct and ``email_verified`` should not be reverting to false, I've tried to replicate the issue but unfortunately could not. Can you please provide a little more information about the setup:
- Are you using postgres?
- What steps did you follow to install SkillTree?
- What configuration parameters are supplied to the skills-service?
- What is the version of the skills-service?
- What steps or actions are being performed in the Dashboard to cause ``email_verified`` column to get reset?

This or any other information you can provide would be greatly appreciated. Thank you!

**Comment by Nomango** [comment]
Thank you for replying! Let me provide more details.

- Yes, I'm using `postgres:11` image
- I installed SkillTree by docker compose with the following yaml:

```yaml
services:
  skilltree:
    container_name: skilltree
    build: ./
    ports:
      - "18080:8080"
    depends_on:
      - postgresql
    environment:
      SKILL_SERVICE_ARGS: " \
        --spring.datasource.url=jdbc:postgresql://postgresql:5432/skills \
        --spring.datasource.username=skills \
        --spring.datasource.password=skillspass \
        --skills.authorization.verifyEmailAddresses=true \
        --skills.config.ui.minNameLength=1 \
        --skills.config.ui.minIdLength=1 \
        "
  postgresql:
    container_name: postgresql
    image: postgres:11
    restart: always
    environment:
      POSTGRES_USER: skills
      POSTGRES_PASSWORD: skillspass
      POSTGRES_DB: skills
      PGDATA: /var/lib/postgresql/data/pgdata
    volumes:
      - ~/docker_volumes/postgres:/var/lib/postgresql/data
    ports:
      - "5432:5432"
```

- SkillTree version: v2.3.3

After I modified 'email_verify=true', I was able to log in for hours. It seems like that it was reset to false after the service restart (not certain).

**Comment by Nomango** [comment]
My Dockerfile:

```dockerfile
FROM openjdk:11

ADD https://github.com/NationalSecurityAgency/skills-service/releases/download/2.3.3/skills-service-2.3.3.jar /root/skills-service.jar

WORKDIR /root

ENV SKILL_SERVICE_ARGS=

CMD java -jar skills-service.jar $SKILL_SERVICE_ARGS
```

**Comment by Nomango** [comment]
Not every service restart will reset it, and I haven't found a pattern of when it changes.

Besides using the frontend provided by Skilltree, I also manually called Skilltree's API to report my skill.

One commonality among the situations I encountered is that in `DBeaver`, when checking `user_attrs`, the modified users always ended up in a middle position (probably because PostgreSQL doesn't sort by ID default).
